src/components/Input/Input.tsx

```diff
@@ -250,7 +250,7 @@ function TextArea({
       size={size}
     >
       <textarea
-        autoComplete={autoComplete && 'autoComplete'}
```

Comment:
@jonthomp here, typescript throws an error saying autoComplete has type `string | undefined`, but we are passing a boolean here. The TextAreaProps interface has autoComplete typed as boolean, but since the autoComplete on textarea says it's string, we can make the type of autoComplete in the interface as string too.
If it has to stay boolean, then we will have to do something like 
`autoComplete={autoComplete ? 'on' : 'off'}` this here. Which one should I do?

Reply:
Hey @YashKandalkar sorry I got halfway through a review and then the fastly outage happened!

I think for `textarea`  `autoComplete={autoComplete ? 'on' : 'off'}` is the right way to go and will keep the interface backwards compatible too

Reply:
Hey, no problem!

Okay I will do that then.